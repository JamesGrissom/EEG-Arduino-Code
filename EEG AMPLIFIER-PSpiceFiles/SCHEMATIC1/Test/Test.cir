** Profile: "SCHEMATIC1-Test"  [ C:\USERS\JAMES\DOCUMENTS\SCHOOL\SPRING 2022\ECE 2804 INTEGRATED DESIGN PROJECT 13616\EEG PROJECT\EEG-ARDUINO-CODE\EEG AMPLIFIER-PSpiceFiles\SCHEMATIC1\Test.sim ] 

** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\cds_spb_home\cdssetup\OrCAD_PSpiceTIPSpice_Install\17.4.0\PSpice.ini file:
.lib "nom_pspti.lib" 
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 1000ns 0 
.OPTIONS ADVCONV
.OPTIONS FILEMODELSEARCH
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
