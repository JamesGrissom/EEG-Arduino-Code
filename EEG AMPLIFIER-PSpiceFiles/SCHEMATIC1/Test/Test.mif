 lib=C:\cds_spb_home\cdssetup\pspTILibDir\INA126.LIB, offset=1948, size=446
 lib=C:\cds_spb_home\cdssetup\pspTILibDir\LF356.LIB, offset=1634, size=1281
 lib=C:\cds_spb_home\cdssetup\pspTILibDir\INA126.LIB, offset=2395, size=1134
 lib=C:\cds_spb_home\cdssetup\pspTILibDir\INA126.LIB, offset=3530, size=866
