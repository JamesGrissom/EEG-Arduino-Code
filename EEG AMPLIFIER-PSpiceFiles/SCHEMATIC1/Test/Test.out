
**** 02/15/22 13:22:16 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-Test"  [ C:\USERS\JAMES\DOCUMENTS\SCHOOL\SPRING 2022\ECE 2804 INTEGRATED DESIGN PROJECT 13616\EEG PROJECT\EE


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\cds_spb_home\cdssetup\OrCAD_PSpiceTIPSpice_Install\17.4.0\PSpice.ini file:
.lib "nom_pspti.lib" 
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 1000ns 0 
.OPTIONS ADVCONV
.OPTIONS FILEMODELSEARCH
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source EEG AMPLIFIER
.EXTERNAL OUTPUT Output
X_U1         N00961 0 N01667 0 0 N00208 N01546 N00954 INA126
X_U2         N00208 N00328 N01546 V- N00346 LF356
X_U3         N00346 N00436 N01546 0 N00452 LF356
X_U4         N00669 N00577 N01546 0 OUTPUT LF356
R_R1         N00961 N00954  5.3k TC=0,0 
R_R2         0 N00328  1k TC=0,0 
R_R3         N00328 N00346  100k TC=0,0 
R_R4         N00436 N00452  100k TC=0,0 
R_R5         0 N00436  1k TC=0,0 
R_R6         N00452 N00641  1k TC=0,0 
R_R7         N00641 N00669  1k TC=0,0 
R_R8         0 N00577  1k TC=0,0 
R_R9         N00577 OUTPUT  1k TC=0,0 
C_C1         N00641 OUTPUT  1n  TC=0,0 
C_C2         0 N00669  1n  TC=0,0 
V_V1         N01546 0 5Vdc
V_V2         N01667 0 DC 0Vdc AC 1Vac 

**** RESUMING Test.cir ****
.END

**** 02/15/22 13:22:16 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-Test"  [ C:\USERS\JAMES\DOCUMENTS\SCHOOL\SPRING 2022\ECE 2804 INTEGRATED DESIGN PROJECT 13616\EEG PROJECT\EE


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U2.DX         X_U3.DX         X_U4.DX         X_U1.x1.dx      
          IS    1.000000E-15    1.000000E-15    1.000000E-15  800.000000E-18 


               X_U1.x2.dx      
          IS  800.000000E-18 


**** 02/15/22 13:22:16 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-Test"  [ C:\USERS\JAMES\DOCUMENTS\SCHOOL\SPRING 2022\ECE 2804 INTEGRATED DESIGN PROJECT 13616\EEG PROJECT\EE


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U1.x1.qx      X_U1.x2.qx      
               PNP             PNP             
       LEVEL    1               1            
          IS  800.000000E-18  800.000000E-18 
          BF  150             150            
          NF    1               1            
          BR    1               1            
          NR    1               1            
         ISS    0               0            
          RE    0               0            
          RC    0               0            
         CJE    0               0            
         VJE     .75             .75         
         CJC    0               0            
         VJC     .75             .75         
         MJC     .33             .33         
        XCJC    1               1            
         CJS    0               0            
         VJS     .75             .75         
          KF    0               0            
          AF    1               1            
          CN    2.2             2.2          
           D     .52             .52         


**** 02/15/22 13:22:16 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-Test"  [ C:\USERS\JAMES\DOCUMENTS\SCHOOL\SPRING 2022\ECE 2804 INTEGRATED DESIGN PROJECT 13616\EEG PROJECT\EE


 ****     Junction FET MODEL PARAMETERS


******************************************************************************




               X_U2.JX         X_U3.JX         X_U4.JX         
               PJF             PJF             PJF             
         VTO   -2              -2              -2            
        BETA   12.500000E-06   12.500000E-06   12.500000E-06 
          IS   30.000000E-12   30.000000E-12   30.000000E-12 


**** 02/15/22 13:22:16 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-Test"  [ C:\USERS\JAMES\DOCUMENTS\SCHOOL\SPRING 2022\ECE 2804 INTEGRATED DESIGN PROJECT 13616\EEG PROJECT\EE


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(   V-)    1.2460  (N00208)     .4114 (N00328)     .1753 (N00346)    3.1212     

(N00436)     .1162 (N00452)    3.0677 (N00577)    1.5547 (N00641)    3.0677     

(N00669)    3.0677 (N00954)     .3805 (N00961)     .3600 (N01546)    5.0000     

(N01667)    0.0000 (OUTPUT)    3.0649 (X_U2.3)     .2934 (X_U2.4)     .5660     

(X_U2.5)     .5381 (X_U2.6)     .7524 (X_U2.7)     .4144 (X_U2.8)    2.3700     

(X_U2.9)    3.1218 (X_U3.3)    1.6187 (X_U3.4)     .5010 (X_U3.5)     .2661     

(X_U3.6) 662.5E-09 (X_U3.7)    3.1242 (X_U3.8)    2.3700 (X_U3.9)    3.0683     

(X_U4.3)    2.3112 (X_U4.4)    1.9222 (X_U4.5)     .9291 (X_U4.6)     .1812     

(X_U4.7)    3.0707 (X_U4.8)    2.3700 (X_U4.9)    3.0951 (X_U2.10)    3.8760    

(X_U2.11)    3.1218                   (X_U2.16)    3.1230                       

(X_U2.17)    3.1230                   (X_U2.21)    5.0000                       

(X_U2.23)    5.6235                   (X_U2.24)    2.8712                       

(X_U2.25)    3.3712                   (X_U2.26)    3.1218                       

(X_U2.27)    3.1218                   (X_U2.49)    3.1230                       

(X_U2.98)    3.1230                   (X_U3.10)    2.6300                       

(X_U3.11)    3.0683                   (X_U3.16)    2.5000                       

(X_U3.17)    2.5000                   (X_U3.21)    5.0000                       

(X_U3.23)    5.6235                   (X_U3.24)    2.8177                       

(X_U3.25)    3.3177                   (X_U3.26)    3.0683                       

(X_U3.27)    3.0683                   (X_U3.49)    2.5000                       

(X_U3.98)    2.5000                   (X_U4.10)    2.6300                       

(X_U4.11)    3.0951                   (X_U4.16)    2.5000                       

(X_U4.17)    2.5000                   (X_U4.21)    5.0000                       

(X_U4.23)    5.7253                   (X_U4.24)    2.8149                       

(X_U4.25)    3.3149                   (X_U4.26)    3.0951                       

(X_U4.27)    3.0951                   (X_U4.49)    2.5000                       

(X_U4.98)    2.5000                   (X_U1.222)     .4114                      

(X_U1.x1.6)     .6055                 (X_U1.x1.7)     .4115                     

(X_U1.x1.8)     .4115                 (X_U1.x1.9)    0.0000                     

(X_U1.x2.6)     .6055                 (X_U1.x2.7)     .4104                     

(X_U1.x2.8)     .4104                 (X_U1.x2.9)    0.0000                     

(X_U1.x1.10)     .6004                (X_U1.x1.11) 102.5E-09                    

(X_U1.x1.12)     .0643                (X_U1.x1.13)     .6004                    

(X_U1.x1.14)     .5883                (X_U1.x1.53)    4.0000                    

(X_U1.x1.54)    1.0000                (X_U1.x1.90)     .0021                    

(X_U1.x1.91)    2.0000                (X_U1.x1.92)   -2.0000                    

(X_U1.x1.99)    2.5000                (X_U1.x2.10)     .6004                    

(X_U1.x2.11) 52.22E-09                (X_U1.x2.12)     .0643                    

(X_U1.x2.13)     .6004                (X_U1.x2.14)     .5883                    

(X_U1.x2.53)    4.0000                (X_U1.x2.54)    1.0000                    

(X_U1.x2.90)    -.0196                (X_U1.x2.91)    2.0000                    

(X_U1.x2.92)   -2.0000                (X_U1.x2.99)    2.5000                




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1        -1.137E-02
    V_V2         4.038E-08
    X_U2.V2     -4.199E-03
    X_U2.V3     -4.627E-03
    X_U2.VA3     0.000E+00
    X_U2.VA7    -6.245E-13
    X_U2.VA8     2.946E-05
    X_U2.V5      1.566E-11
    X_U2.V4      1.638E-11
    X_U3.V2     -5.322E-04
    X_U3.V3      4.393E-13
    X_U3.VA3     0.000E+00
    X_U3.VA7    -6.245E-13
    X_U3.VA8     2.952E-05
    X_U3.V5      1.566E-11
    X_U3.V4      1.638E-11
    X_U4.V2     -1.496E-03
    X_U4.V3      4.661E-13
    X_U4.VA3     0.000E+00
    X_U4.VA7    -7.263E-13
    X_U4.VA8     1.510E-03
    X_U4.V5      5.124E-12
    X_U4.V4      5.098E-11
    X_U1.x1.vb   6.055E-06
    X_U1.x1.vc   3.589E-12
    X_U1.x1.ve  -6.117E-06
    X_U1.x1.vlim   2.111E-06
    X_U1.x1.vlp  -1.999E-12
    X_U1.x1.vln  -2.003E-12
    X_U1.x2.vb   6.055E-06
    X_U1.x2.vc   3.589E-12
    X_U1.x2.ve  -6.117E-06
    X_U1.x2.vlim  -1.957E-05
    X_U1.x2.vlp  -2.020E-12
    X_U1.x2.vln  -1.981E-12

    TOTAL POWER DISSIPATION   8.54E-02  WATTS



          JOB CONCLUDED

**** 02/15/22 13:22:16 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "SCHEMATIC1-Test"  [ C:\USERS\JAMES\DOCUMENTS\SCHOOL\SPRING 2022\ECE 2804 INTEGRATED DESIGN PROJECT 13616\EEG PROJECT\EE


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  License check-out time            =       29.09
  Total job time (using Solver 1)   =        1.64
